### PR TITLE
Fix for ReadSecurity not being serialized

### PR DESCRIPTION
**Pull request by erwinvanhunen**

### CODE REVIEW

File: Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/PnPObjectsMapper.cs [hunk 1]
```diff
@@ -194,6 +194,7 @@ public static void MapProperties(Object source, Object destination, Dictionary<S
                                     {
                                         sourceValue = null;
                                     }
+                                    spSpecified.SetValue(destination, isSpecified);
```
Comment (PaoloPia) [inline Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/PnPObjectsMapper.cs]: I think this one is wrong.

File: Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/Serializers/V201705/ListInstancesSerializer.cs [hunk 2]
```diff
@@ -168,6 +168,7 @@ public override void Serialize(ProvisioningTemplate template, object persistence
                 resolvers.Add($"{listInstanceType}.DraftVersionVisibilitySpecified", new ExpressionValueResolver(() => true));
                 resolvers.Add($"{listInstanceType}.MaxVersionLimitSpecified", new ExpressionValueResolver(() => true));
                 resolvers.Add($"{listInstanceType}.MinorVersionLimitSpecified", new ExpressionValueResolver(() => true));
+                resolvers.Add($"{listInstanceType}.ReadSecuritySpecified", new ExpressionValueResolver(() => true));
```
Comment (PaoloPia) [inline Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/Serializers/V201705/ListInstancesSerializer.cs]: This is ok, and I can place it in my own branch.

File: Core/OfficeDevPnP.Core/Pages/ClientSidePageCanvas.cs [hunk 3]
```diff
@@ -1172,7 +1172,11 @@ private void LoadFromHtml(string html)
                     }
                     else if (controlType == typeof(ClientSideWebPart))
                     {
-                        var control = new ClientSideWebPart();
+                        var control = new ClientSideWebPart()
```
Comment (PaoloPia) [inline Core/OfficeDevPnP.Core/Pages/ClientSidePageCanvas.cs]: Changes to this file are not related to the ReadSecurity issue. Why are they here?

Comment (erwinvanhunen) [reply Core/OfficeDevPnP.Core/Pages/ClientSidePageCanvas.cs]: Not related, but important to get in. Just a mixup of PRs

File: Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/PnPObjectsMapper.cs (hunk 1 above)
Comment (erwinvanhunen) [reply Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/PnPObjectsMapper.cs]: I had to add that one to make it work over here...
